_layouts/team-page.html

```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+---
+
+<div class="team-profile">
+    <div class="row">
+        <div class="col">
+            {% if page.img %}
+            <img class="profile-img" src="{{ site.baseurl }}/img/team/{{ page.img }}" alt="picture of {{ page.name }}">
+            {% else %}
+            <img class="profile-img" src="https://api.adorable.io/avatars/512/{{ page.name }}.png" alt="temporary picture"/>
+            {% endif %}
+        </div>
+        
+        <div class="col">
+            <h1 class="title">{{ page.firstname }} {{ page.lastname }}</h1>
+            <p class="no-margin text-15x">{{ page.title }}</p>
+
+            {% if page.links or page.email %}
+            <div class="team-links">
+                <ul>
+                    {% for link in page.links %}
+                    <li><a class="tla-link" href="{{ link.href }}" target="_blank">{{ link.name }}</a></li>
+                    {% endfor %}
+                    {% if page.email %}
+                    <li><a class="tla-link" href="mailto:{{ page.email }}">{{ page.email }}</a></li>
+                    {% endif %}
+                </ul>
+            </div>
+            {% endif %}
+        </div>
+    </div>
+</div>
+
+{% if page.content %}
+<div class="team-content">
+    {{ page.content }}
+</div>
+{% endif %}
```

Comment:
Do you want to add a trailing button that says "meet more peeps on the team" or something?